src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj

```diff
@@ -118,6 +118,7 @@
 	
 	<ItemGroup>
 	  <Folder Include="MacOS\" />
+	  <Folder Include="obj\" />
```

Comment:
Is this needed?

Reply:
No, but I'll remove it in the next PR. Thanks!